.zsh/exports.zsh

```diff
@@ -87,3 +87,6 @@ unset mysql_credentials_path
 
 # Export camlistore's secret keyring
 # [[ -x "$(brew --prefix)/bin/camput" ]] && export CAMLI_SECRET_RING="${HOME}/.gnupg/secring.gpg"
+
+# Docker Environment Variable export
+eval $(docker-machine env)
```

Comment:
docker-machine is not available everywhere (for example on Linux where I also run this config). Change this to:

```shell
if [[ -x $(which docker-machine 2> /dev/null ]]; then
  eval $(docker-machine env)
fi